tests/draining/queue_test.py

```diff
@@ -479,6 +480,9 @@ def test_host_from_instance_id():
         mock_ec2_describe.return_value = [{"InstanceId": "i-123"}]
         assert host_from_instance_id(sender="aws", receipt_handle="rcpt", instance_id="i-123",) is None
 
+        mock_ec2_describe.side_effect = botocore.exceptions.ClientError({}, "")
```

Comment:
might be nice to add a comment here like the one above describe the case we're testing here